### PR TITLE
Add missing PropTypes and improve consistency

### PR DESCRIPTION
**Pull request by lucafalasco**
#### :rocket: Enhancements

- Add missing PropTypes and try to enhance consistency among packages

### CODE REVIEW

File: packages/vx-brush/src/brushes/BoxBrush.js [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
-export default props => {
-  const {
-    brush,
-    className,
-    fill = 'rgba(102, 181, 245, 0.1)',
-    stroke = 'rgba(102, 181, 245, 1)',
-    strokeWidth = 1,
-    ...otherProps
-  } = props;
+BoxPlot.propTypes = {
```
Comment (hshoff) [inline packages/vx-brush/src/brushes/BoxBrush.js]: `BoxPlot` => `BoxBrush`

File: packages/vx-brush/src/brushes/BoxBrush.js [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
+  brush: PropTypes.object,
+  className: PropTypes.string,
+  fill: PropTypes.string,
+  stroke: PropTypes.string,
+  strokeWidth: PropTypes.number
+};
+
+export default function BoxPlot({
```
Comment (hshoff) [inline packages/vx-brush/src/brushes/BoxBrush.js]: `BoxPlot` => `BoxBrush`

File: packages/vx-clip-path/package.json [hunk 3]
```diff
@@ -89,5 +89,8 @@
       "raf/polyfill",
       "<rootDir>/test/enzyme-setup.js"
     ]
+  },
+  "dependencies": {
+    "prop-types": "^15.6.2"
```
Comment (hshoff) [inline packages/vx-clip-path/package.json]: can remove this as it's already included https://github.com/hshoff/vx/pull/392/files#diff-8b568cc49c33b4133a4de279448cea60R38

File: packages/vx-group/src/Group.js [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import PropTypes from 'prop-types';
```
Comment (hshoff) [inline packages/vx-group/src/Group.js]: Can remove this as it is already imported

File: packages/vx-group/src/Group.js [hunk 5]
```diff
@@ -10,6 +11,14 @@ Group.propTypes = {
...
 };
 
+Group.propTypes = {
+  children: PropTypes.any,
+  className: PropTypes.string,
+  top: PropTypes.number,
+  left: PropTypes.number,
+  transform: PropTypes.string
+};
+
```
Comment (hshoff) [inline packages/vx-group/src/Group.js]: Can remove this as it's already included

File: packages/vx-threshold/src/Threshold.js [hunk 6]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import cx from 'classnames';
 import PropTypes from 'prop-types';
 import { Area } from '@vx/shape';
+import PropTypes from 'prop-types';
```
Comment (hshoff) [inline packages/vx-threshold/src/Threshold.js]: Can remove this as it's already imported

**Comment by lucafalasco** [comment]
@hshoff sorry about that, I think I messed things up when I used Github merge tool 😅

**Comment by williaster** [comment]
closing this since we generate proptypes from TS now

**Pull request closed without merging**